adv-rumors.lua

```diff
@@ -80,52 +84,36 @@ function addKeyword(choice, keyword)
     choice.key_word:insert('#', keyword_ptr)
 end
 
+function addKeywords(choice, keywords)
+    for i, keyword in ipairs(keywords) do
+        addKeyword(choice, keyword)
+    end
+end
+
 function addKeywordsForChoice(choice)
     local fulltext = choiceToString(choice)
 
     -- Special cases
     if string.find(fulltext, "slew") or string.find(fulltext, "slain") then
-        addKeyword(choice, 'slew')
         addKeyword(choice, 'slay')
```

Comment:
the previous code chose `slew`. I don't have a particular preference, but I'd like to understand what your reasoning is for the change.

Reply:
Slew/slain refer to different kill events (you slew/was slain by), these key words will be added automatically by code below

Meanwhile, "slay" is a general key word given to both events as a scored kill.
Now that I think about it "kill" should also be a key word maybe